src/workerd/jsg/jsg.c++

```diff
@@ -418,7 +410,7 @@ void ExternalMemoryAdjustment::maybeDeferAdjustment(ssize_t amount) {
 }
 
 ExternalMemoryAdjustment::ExternalMemoryAdjustment(
-    kj::Own<ExternalMemoryTarget> externalMemory, size_t amount)
+    kj::Own<const ExternalMemoryTarget> externalMemory, size_t amount)
```

Comment:
```suggestion
    kj::Own<const ExternalMemoryTarget>&& externalMemory, size_t amount)
```

I guess?

Reply:
These are mostly functionally equivalent, but the `&&` adds noise, and sometimes can be confusing if the called function doesn't actually move away the value. Most of the time I think it's best not to declare parameters as rvalue references.